container-images/mds-provider/event-processor.ts

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
I don't think we're quite ready for `event-processor` yet. I would remove this file.

Reply:
removed